src/Nethermind/Nethermind.Runner/configs/ndm_consumer.cfg

```diff
@@ -1,36 +1,24 @@
 {
   "Init": {
     "EnableUnsecuredDevWallet": true,
-    "KeepDevWalletInMemory": true,
+    "KeepDevWalletInMemory": false,
     "WebSocketsEnabled": true,
-    "NetworkEnabled": true,
-    "DiscoveryEnabled": false,
+    "StoreReceipts": true,
```

Comment:
Did you add tests for it?

Reply:
Tests for? It was just an updated config, similarly to `ndm_consumer_goerli.cfg` etc.